src/pages/trade/ui/positions/index.tsx

```diff
@@ -136,7 +137,9 @@ const Positions = observer(({ showInactive }: { showInactive: boolean }) => {
                           {position.orders.map((order, i) => (
                             <ValueViewComponent
                               key={i}
-                              valueView={order.amount}
+                              valueView={
+                                position.isClosed && i === 1 ? order.basePrice : order.amount
```

Comment:
For positions that are closed, but have 2 orders, we show the basePrice for the 2nd order.